tests/test_errors.py

```diff
@@ -47,6 +47,8 @@ class TestErrorsTranslation(TestCase):
 
     def setUp(self):
         self.config = testing.setUp()
+        settings = self.config.get_settings()
+        settings["available_languages"] = "en fr"
```

Comment:
you could probably use `config.add_settings()` https://docs.pylonsproject.org/projects/pyramid/en/latest/api/config.html#pyramid.config.Configurator.add_settings

Reply:
Sure.